_docs/src/documentation/rex_pipeline.md

```diff
@@ -89,11 +89,11 @@ Also known as combi builds and jumbo builds.
 
 Note: this has nothing to do with the Unity game engine.
 
-A unity build allows the build system to merge source files together, creating 1 big source files. This is simply done with C++ include statements including different .cpp files into intermediate .cpp files. 
+A unity build allows the build system to merge source files together, creating 1 big source file. This is simply done with C++ include statements including different .cpp files into intermediate .cpp files. 
 
 These intermediate files are then passed on to the compiler which improves compilation times a lot as the compiler doesn't have to open as many files and can parse more files in 1 go.
 
-Note: unity build support is only for Visual Studio generated projects, it's currently not supported for Ninja builds
+Note: Files modified on disk, different from their repo equivalent are excluded from unity files to improve compilation. They also have optimizations disabled so it's easier to debug them.
```

Comment:
INFO: Does this mean that the code that I change locally is not included within the unity build? For instance I was developing the renderer so all the code that was included with that feature is not included within the unity build?